ftp-edi-message-to-salesforce-opportunity/README.md

```diff
@@ -0,0 +1,82 @@
+# FTP B2B EDI message to Salesforce Opportunity
+
+This sample reads EDI files from a given FTP location, converts those EDI messages to Ballerina records, and creates a Salesforce Opportunity for each EDI message.
```

Comment:
```suggestion
This sample reads EDI files from a given FTP location, converts those EDI messages to Ballerina records, and creates a Salesforce opportunity for each EDI message.
```

Reply:
It seems in general they use lowercase o for this. check with Salesforce docs and update in all the places.